coldfront/templates/email/allocation_revoked.txt

```diff
@@ -0,0 +1,10 @@
+Dear {{center_name}} user,
+
+Your allocation request for {{resource}} has been revoked.
+
+Please login to view a message from the system administrators: {{allocation_url}}
+If you are a student or collaborator under this project, you are receiving this notice as a courtesy. If you would like
```

Comment:
Add a new line in between these two paragraphs.  I will update the other email templates this way too